src/program/media/item.py

```diff
@@ -321,20 +325,20 @@ def get_aliases(self) -> dict:
     def __hash__(self):
         return hash(self._id)
 
-    def reset(self, soft_reset: bool = False):
+    def reset(self):
         """Reset item attributes."""
         if self.type == "show":
             for season in self.seasons:
                 for episode in season.episodes:
-                    episode._reset(soft_reset)
-                season._reset(soft_reset)
+                    episode._reset()
+                season._reset()
         elif self.type == "season":
             for episode in self.episodes:
-                episode._reset(soft_reset)
-        self._reset(soft_reset)
-        self.store_state()
+                episode._reset()
+        self._reset()
+        self.store_state(States.Indexed)
```

Comment:
might be dangerous here to go Indexed on a reset, should we check first if the item has a title, then if so use Indexed, otherwise, requested?

Reply:
Yes, good point!